src/App.tsx

```diff
@@ -4,11 +4,13 @@ import { Login } from "./component/Login";
 import { SignUp } from "./component/SignUp";
 import { ForgotPassword } from "./component/ForgotPassword";
 import { Profile } from "./component/Profile";
+import { Settings } from "./component/Settings";
```

Comment:
This component is not used, and it seems to be a duplicate (or similar) to Setting.

Reply:
`Settings` is the original setting page @KeN095 wrote, but I refactored the code and added the ability to read/write uuid.

Reply:
> `Settings` is the original setting page @KeN095 wrote, but I refactored the code and added the ability to read/write uuid. What you think of the code @KeN095

It's a lot more clean compared to my implementation, @chungwwei. Wouldn't have realized separating `EmailPane` and `PasswordPane.tsx` into their own components. I did wanted to see the preview URL to see it working but yeah, my implementation can be removed.